core/src/test/java/org/apache/iceberg/rest/TestResourcePaths.java

```diff
@@ -179,4 +179,60 @@ public void viewWithMultipartNamespace() {
     assertThat(withPrefix.view(ident)).isEqualTo("v1/ws/catalog/namespaces/n%1Fs/views/view-name");
     assertThat(withoutPrefix.view(ident)).isEqualTo("v1/namespaces/n%1Fs/views/view-name");
   }
+
+  @Test
+  public void planEndpointPath() {
+    TableIdentifier tableId = TableIdentifier.of("test_namespace", "test_table");
+    ResourcePaths paths = new ResourcePaths("test-prefix");
+
+    String submitPlanPath = paths.planTableScan(tableId);
+
+    assertThat(submitPlanPath)
+        .isEqualTo("v1/test-prefix/namespaces/test_namespace/tables/test_table/plan");
+
+    // Test with different identifiers
+    TableIdentifier complexId = TableIdentifier.of(Namespace.of("db", "schema"), "my_table");
+    String complexPath = paths.planTableScan(complexId);
+
+    assertThat(complexPath).isEqualTo("v1/test-prefix/namespaces/db%1Fschema/tables/my_table/plan");
+  }
+
+  @Test
+  public void fetchScanTasks() {
```

Comment:
```suggestion
  public void fetchScanTasksPath() {
```
to align with the naming of the other server-side planning test names